docs/getting-started/v3-migration.md

```diff
@@ -182,7 +182,6 @@ Animation system was completely rewritten in Chart.js v3. Each property can now
 * `helpers.toDegrees` was renamed to `helpers.math.toDegrees`
 * `helpers.toRadians` was renamed to `helpers.math.toRadians`
 * `Scale.calculateTickRotation` was renamed to `Scale.calculateLabelRotation`
-* `TimeScale.getLabelCapacity` was renamed to `TimeScale._getLabelCapacity`
```

Comment:
This should probably still be present, but maybe moved to the removed section above like the other APIs that were make private? Same with `tickFormatFunction`

Reply:
Oh wait, I didn't notice that it was renamed. The title of this PR might not be very accurate :-)

Reply:
Yeah, did not initially notice the changes needed for ts 3.8

Reply:
Ah, I see. It's enforcing the access modifiers now. I didn't realize that

Maybe we should move `getPixelForOffset` and `tickFormatFunction` above to the removed section and mention that they're now private? We do that for some other methods so trying to be consistent